lib/ddtrace/context.rb

```diff
@@ -149,17 +161,50 @@ def get
         trace = @trace
         sampled = @sampled
 
-        attach_sampling_priority if sampled && @sampling_priority
-        attach_origin if @origin
-
         # still return sampled attribute, even if context is not finished
         return nil, sampled unless check_finished_spans()
```

Comment:
Above in the comment it says `It returns +[ni,nil]+ if the Context is not finished`